src/sidebar/components/search/FilterControls.tsx

```diff
@@ -26,19 +40,46 @@ type FilterToggleProps = {
  */
 function FilterToggle({
   label,
+  icon,
   description,
   active,
   setActive,
   testId,
 }: FilterToggleProps) {
+  const IconComponent = icon;
```

Comment:
Perhaps you can alias it inline when unpacking the props. If I remember correctly, we use that pattern frequently in frontend-shared.

```suggestion
  icon: IconComponent,
  description,
  active,
  setActive,
  testId,
}: FilterToggleProps) {
```